src/lib/cypress-testrail-reporter.ts

```diff
@@ -68,7 +68,9 @@ export class CypressTestRailReporter extends reporters.Spec {
         return;
       }
 
+      // publish test cases results & close the run
       this.testRail.publishResults(this.results);
+      this.testRail.closeRun(this.results);
```

Comment:
We should send request to close run after publish is complete. Use `then` after `publishResults` function

Reply:
Right, in fact, I was running into an intermittent error that says Action cannot be performed. Test run already marked as completed.

Fixed now. Let me know if all is good for the merge. Thanks!